app/user.go

```diff
@@ -23,7 +26,10 @@ func init() {
 // NewUser creates a new user
 func NewUser() *User {
 	return &User{
-		Wallet:    blockchain.NewWallet(),
+		HotWallet: HotWallet{
+			Wallet: blockchain.NewWallet(),
```

Comment:
Does this work? It seems like it shouldn't because HotWallet has no member "Wallet".